weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooks.ts

```diff
@@ -1948,16 +1948,22 @@ export const traceCallStatusCode = (
 
 export const traceCallLatencyS = (
   traceCall: traceServerTypes.TraceCallSchema
+) => {
+  return traceCallLatencyMs(traceCall) / 1000;
+};
+
+export const traceCallLatencyMs = (
+  traceCall: traceServerTypes.TraceCallSchema
 ) => {
   const startDate = convertISOToDate(traceCall.started_at);
   const endDate = traceCall.ended_at
     ? convertISOToDate(traceCall.ended_at)
     : null;
-  let latencyS = 0;
+  let latencyMs = 0;
   if (startDate && endDate) {
-    latencyS = (endDate.getTime() - startDate.getTime()) / 1000;
+    latencyMs = endDate.getTime() - startDate.getTime();
```

Comment:
Optional, I think it is a little cleaner to just return here or return 0 at end.